baseframe/forms/widgets.py

```diff
@@ -286,3 +289,56 @@ def __call__(self, field, **kwargs):
             )
         html.append('</%s>' % self.html_tag)
         return Markup('\n'.join(html))
+
+
+class ImgeeWidget(wtforms.widgets.Input):
+    input_type = 'hidden'
+
+    def __call__(self, field, **kwargs):
+        id_ = kwargs.pop('id', field.id)
+        kwargs.setdefault('type', self.input_type)
+        imgee_host = current_app.config.get('IMGEE_HOST')
+        if not imgee_host:
+            raise ValueError("No imgee server specified")
```

Comment:
"No Imgee server specified in config var IMGEE_HOST" (so that the dev knows what var to fix without looking at this source)